packages/components/src/components/get-your-api-game-right-section.tsx

```diff
@@ -0,0 +1,105 @@
+import { cn } from '../cn';
+import { CallToAction } from './call-to-action';
+import { DecorationIsolation } from './decorations';
+import { Heading } from './heading';
+
+export function GetYourAPIGameRightSection({ className }: { className?: string }) {
+  return (
+    <section
+      className={cn(className, 'relative overflow-hidden rounded-3xl bg-primary md:h-[308px]')}
+    >
+      <DecorationIsolation>
+        <GreenArchDecoration className="absolute inset-y-0 right-0 hidden opacity-10 md:block [@media(min-width:1300px)]:opacity-100" />
+        <StrokeDecoration className="absolute right-[607px] max-md:right-[-36px] max-md:top-[-71px] max-md:size-[200px] max-md:rotate-180 md:bottom-0" />
+      </DecorationIsolation>
+      <div className="flex flex-col items-stretch justify-between gap-x-6 gap-y-4 p-4 sm:items-center md:h-[308px] md:flex-row md:px-24">
+        <Heading
+          as="h2"
+          size="md"
+          className="text-[40px] leading-[1.2] tracking-[-0.2px] max-sm:text-balance max-sm:text-center md:text-[56px] md:leading-[1.142586] md:tracking-[-0.56px]"
+        >
+          Get your API game right.
+        </Heading>
+        <div className="flex flex-col gap-x-4 gap-y-2 whitespace-pre sm:flex-row">
+          <CallToAction variant="secondary-inverted" href="https://app.graphql-hive.com/">
+            Get started for free
+          </CallToAction>
+          <CallToAction
+            variant="tertiary"
+            href="https://the-guild.dev/contact"
+            onClick={event => {
+              if (typeof window !== 'undefined' && '$crisp' in window) {
+                (window.$crisp as { push(cmd: string[]): void }).push(['do', 'chat:open']);
+                event.preventDefault();
+              }
```

Comment:
```suggestion
                (window.$crisp as { push(cmd: string[]): void }).push(['do', 'chat:open']);
                event.preventDefault();
```

onClick is not available on server